app/models/reviewer_access.rb

```diff
@@ -0,0 +1,94 @@
+# manage anonymous access to private studies via access code & pin
+# will generate a short-lived session for reviewers with valid credentials
+class ReviewerAccess
+  include Mongoid::Document
+  include Mongoid::Timestamps
+  include ActiveSupport::SecurityUtils
+
+  field :access_code, type: String # UUID
+  field :pin, type: String # 6 character string
```

Comment:
Length was changed from 6 to 10, so let's update this comment for consistency.

```suggestion
  field :pin, type: String # 10 character string
```

Reply:
446268b